src/test/tables/tpch/minimal/customer.tbl

```diff
@@ -0,0 +1,3 @@
+c_custkey|c_name|c_address|c_nationkey|c_phone|c_acctbal|c_mktsegment|c_comment
+int|string|string|int|string|float|string|string
+0|Max Mustermann|Berlin|0|555-123-4567|10.0|BUILDING|-
```

Comment:
I believe we should not commit any generated files, only the steps necessary to generate those files. We will need dbgen/qgen as a submodule later anyway.

Reply:
I completely agree, dbgen should generate these on the fly (yet they should remain on disk, so the dev can have a look at them). 
We should decide about this in a separate issue and leave the generated files in for now: #490

Reply:
ok